tests/chain_sync_test.rs

```diff
@@ -143,17 +143,40 @@ mod chain_sync_test {
         assert_eq!(node.get_node_info().num_channels, 0);
     }
 
+    #[test]
+    fn test_multiple_txs_simultaneously() {
+        let node_handle = setup();
+        let node = node_handle.start().unwrap();
+        let node_id = node.get_node_info().node_pubkey.to_hex();
+
+        // open 5 channels and force-close 3 of them right away
+        let mut open_channels = open_5_chans_close_2(&node_id);
+
+        // mine a block and do the same again and remove 1 of the previously opened channels
+        nigiri::try_cmd_repeatedly(nigiri::mine_blocks, 1, 10, HALF_SEC).unwrap();
+        let _ = open_5_chans_close_2(&node_id);
+        nigiri::lnd_force_close_channel(open_channels.pop().unwrap()).unwrap();
+
+        nigiri::try_cmd_repeatedly(nigiri::mine_blocks, 10, 10, HALF_SEC).unwrap();
+        sleep(Duration::from_secs(10));
+
+        assert_eq!(node.get_node_info().num_channels, 3);
```

Comment:
You can also check here for the number of usable channels. It should be 3 as well.

Reply:
Yes, thanks for that great input. Fixed here: d0a7e80099ed9a2e53c1ef793612bd902857d03f